### PR TITLE
New styling

### PR DESCRIPTION
**Pull request by larisaescobar**

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <header class="header">
     <div class="header-title">
-      <img src="{{ "/img/mews-logo-small.png" | prepend: site.baseurl }}" />
```
Comment (jakubtkadlecek) [inline _includes/header.html]: I do not see deletion of this file. Do we need it still?

File: _sass/_layout.scss [hunk 2]
```diff
@@ -52,19 +62,25 @@
                 line-height: 23px;
             }
         }
-
+
```
Comment (jakubtkadlecek) [inline _sass/_layout.scss]: unused whitespace

File: _sass/_layout.scss [hunk 3]
```diff
@@ -155,7 +173,11 @@ nav {
       font-size: 18px;
       position: absolute;
       top: 3px;
-      left: 8px;
+      right: 8px;
+
+      &.up {
+      transform: rotate(-180deg);
```
Comment (jakubtkadlecek) [inline _sass/_layout.scss]: missing indentation

File: _sass/_layout.scss [hunk 4]
```diff
@@ -245,3 +267,22 @@ nav {
     }
   }
 }
+
+.chevron::before {
```
Comment (jakubtkadlecek) [inline _sass/_layout.scss]: indentations please

File: _includes/header.html (hunk 1 above)
Comment (larisaescobar) [inline _includes/header.html]: I don't think we need it, I'll remove it.

**Pull request merged**